### PR TITLE
Add a basic MCF parser in Java

### PR DESCRIPTION
**Pull request by pradh**
* Initial check-in of the basic MCFGraph proto and very basic parsing of Instance MCF, and tests.
* Set up maven build files (following https://github.com/neo4j/neo4j structure), and ensure it is importable into IntelliJ
* Add Vocabulary static class, and confirm parity with internal implementation.

### CODE REVIEW

**Review by shifucun — COMMENTED** [review]
Should add license header for most of the code files

File: README.md [hunk 1]
```diff
@@ -11,11 +11,17 @@ includes [APIs](https://docs.datacommons.org/api/) and visual tools to easily
 explore and analyze data across different datasets without data cleaning or
 joining.
 
-## License
+## Dependencies
 
-Apache 2.0
+* The tools are built using Apache Maven version 3.8.0.
+
+  ```brew install maven```
```
Comment (shifucun) [inline README.md:18]: can mention this is for Mac?

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+// TODO: Implement COMPLEX_VALUE parsing
+// TODO: Implement TEMPLATE_MCF parsing
+public class McfParser {
```
Comment (shifucun) [inline util/src/main/java/org/datacommons/util/McfParser.java:21]: Add comments for class and public methods?

File: util/src/main/java/org/datacommons/util/McfParser.java [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+    McfParser parser = McfParser.init(Mcf.McfType.INSTANCE_MCF, isResolved);
+    for (String l : mcf_string.split("\n")) {
+      parser.parseLine(l);
+    }
+    return parser.finish();
+  }
+
+  public static McfGraph parseInstanceMcfFile(String file_name, boolean isResolved) throws IOException {
+    McfParser parser = McfParser.init(Mcf.McfType.INSTANCE_MCF, isResolved);
+    Stream<String> lines = Files.lines(FileSystems.getDefault().getPath(file_name), StandardCharsets.UTF_8);
```
Comment (shifucun) [inline util/src/main/java/org/datacommons/util/McfParser.java:49]: nit: enable line length check (auto wrap) in IDE?

File: util/src/test/java/org/datacommons/util/MCFParserTest.java [hunk 4]
```diff
@@ -0,0 +1,123 @@
+package org.datacommons.util;
```
Comment (shifucun) [inline util/src/test/java/org/datacommons/util/MCFParserTest.java:1]: would be nice to add cloudbuild for testing

File: util/src/main/java/org/datacommons/util/McfParser.java (hunk 2 above)
Comment (pradh) [reply util/src/main/java/org/datacommons/util/McfParser.java:36]: Done.

**Review by pradh — COMMENTED** [review]
> Should add license header for most of the code files

Thanks for reminding... done.

**Comment by pradh** [comment]
Thanks for the review!

File: util/src/main/java/org/datacommons/util/Vocabulary.java [hunk 5]
```diff
@@ -0,0 +1,168 @@
+// Copyright 2019 Google LLC
```
Comment (shifucun) [inline util/src/main/java/org/datacommons/util/Vocabulary.java:1]: nit: 2021 and for other files.

**Review by shifucun — APPROVED** [review]

File: util/src/main/java/org/datacommons/util/Vocabulary.java (hunk 5 above)
Comment (pradh) [reply util/src/main/java/org/datacommons/util/Vocabulary.java]: Done.

**Pull request merged**